src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -158,9 +158,11 @@ def _server_update_custom_func(instance,
     module = import_module(server_module_path.replace('server', 'server_update_parameters'))
     ServerUpdateParameters = getattr(module, 'ServerUpdateParameters')
 
-    if capacity is not None:
-        instance.sku.name = _get_sku_name(instance.sku.tier, instance.sku.family, capacity)
-        instance.sku.capacity = capacity
+    if sku_name is not None:
```

Comment:
Why not just 
`if sku_name:`?

Reply:
sku has additional properties like capacity. Thats why it is specified sku.name

Reply:
No. My question is why not just test if the `sku_name` is `None` simply using `if sku_name`

Reply:
https://www.udacity.com/wiki/cs258/truthiness-in-python

Reply:
fixed